addons/udes_priorities/models/stock_picking.py

```diff
@@ -10,9 +10,12 @@ def _default_priority(self):
         return self.env.ref("udes_priorities.normal").reference
 
     priority = fields.Selection(selection="get_priorities_for_selection", default=_default_priority)
+    u_priority_id = fields.Many2one(comodel_name="udes_priorities.priority")
```

Comment:
Is u_priority_id field used? I think only  u_priority_sequence is used, and can set that field on def create or def write in the same way you are doing for u_priority_id, or dont need to have it computed or related, maybe can set the value of u_priority_sequence in create and write

Reply:
Ignore my suggestion, from discussions had on channel